src/main/java/com/amihaiemil/docker/Version.java

```diff
@@ -0,0 +1,11 @@
+package com.amihaiemil.docker;
+
+import javax.json.JsonObject;
+
+/**
+ * Version API.
+ * @author Michael Lux (michi.lux@gmail.com)
+ * @since 0.0.11
+ */
+public interface Version extends JsonObject {
+}
```

Comment:
@milux We should also have accesor methods, since the JsonObject has quite a few attributes, as I see.
Also, it would be nice to have the ``docker()`` method as well, which returns the parent ``Docker`` instance.

If you already spent too much time, please leave a puzzle for continuing implementation.

Reply:
Done, plus I have no clue what those "puzzles" are? Read about it somewhere in your code, but didn't quite get it. Can you provide an example for better understanding?

Reply:
Removed the local variables because I think they're pointless anyway. Can undo this if you think differently.

Reply:
@milux Can you remove the ``get`` and ``is`` prefixes from the methods name? I never use them, they are just noise in the code :) 

I also don't see the ``docker()`` method, can you add it?